src/dialogs/vkbasalt.py

```diff
@@ -0,0 +1,326 @@
+# vkbasalt.py
+#
+# Copyright 2022 Hari Rana / TheEvilSkeleton <theevilskeleton@riseup.net>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+Terminologies:
+--------------
+cas: Contrast Adaptive Sharpening
+dls: Denoised Luma Sharpening
+fxaa: Fast Approximate Anti-Aliasing
+smaa: Subpixel Morphological Anti-Aliasing
+clut (or lut): Color LookUp Table
+'''
+
+import os
+from gi.repository import Gtk, GLib, Adw, Gdk
+from bottles.backend.utils.vkbasalt import parse, ParseConfig
+from bottles.backend.utils.manager import ManagerUtils
+from bottles.dialogs.filechooser import FileChooser  # pyright: reportMissingImports=false
+import logging
+
+class vkbasalt_settings:
```

Comment:
classes should be PascalCase

Reply:
@A6GibKm hey, you suggested to make this change. What are your thoughts on changing it back?

Reply:
I think this was a communication issue, yes PascalCase for all classes, always.

Reply:
ugh, I thought I had replied to this already. Yeah all classes are PascalCase, *always*, maybe there was a communication issue somewhere.